### ISSUE TITLE
Error message for mithril-client snapshot download

### ISSUE THREAD

**Issue by TrevorBenson** [input-output-hk/mithril#1375]
## Context & versions
When trying to download the latest snapshot on the cardano mainnet an error is received about missing field verification_key.

Example:
```
1/6 - Checking local disk info…
2/6 - Fetching the certificate's information…
Nov 28 16:54:35.865 CRIT Could not create certificate from API message: missing field `verification_key` at line 1 column 503.
Error: Snapshot Service can not get the certificate for id: '023c3aed434b973b21981017510a473683f07b779ba0f870a7693d69f5cdae5c'

Caused by:
    missing field `verification_key` at line 1 column 503
```

Version
```
mithril-client --version
mithril-client 0.4.12+e5ae362
```

Mithril Config
```
NETWORK=mainnet
RELEASE=release
AGGREGATOR_ENDPOINT=https://aggregator.release-mainnet.api.mithril.network/aggregator
DB_DIRECTORY=/opt/cardano/cnode/db
GENESIS_VERIFICATION_KEY=5b3139312c36362c3134302c3138352c3133382c31312c3233372c3230372c3235302c3134342c32372c322c3138382c33302c31322c38312c3135352c3230342c31302c3137392c37352c32332c3133382c3139362c3231372c352c31342c32302c35372c37392c33392c3137365d
SNAPSHOT_DIGEST=latest
```

## Steps to reproduce
  1. Obtained mithril-client from binary release v2342.0
  2. Start the mithril-client snapshow download in verbose mode `mithril-client -v --aggregator-endpoint ${AGGREGATOR_ENDPOINT} snapshot download --download-dir ${DB_DIRECTORY} latest`
  3. Observe error

## Actual behavior
Always receives a missing field of the snapshot certificate, fails to download snapshot and extract from archive.

## Expected behavior
Snapshot download completes and extracts a valid cardano mainnet database.

### TRACE level verbosity

Available in [this gist](https://gist.github.com/TrevorBenson/ac5fe32f0934bcd5181db3b6c097fc3b)

I see both **next_aggregate_verification_key** and **aggregate_verification_key** in the json, but no **verification_key**.

**Comment by jpraynaud** [comment]
Hi @TrevorBenson, you just need to use the newest `mithril-client` released in [`2347.0`](https://github.com/input-output-hk/mithril/releases/tag/2347.0) in order to restore the snapshot (the minimum version is now `0.5.5`) :+1:

**Comment by TrevorBenson** [comment]
> Hi @TrevorBenson, you just need to use the newest `mithril-client` released in [`2347.0`](https://github.com/input-output-hk/mithril/releases/tag/2347.0) in order to restore the snapshot (the minimum version is now `0.5.5`) 👍

@jpraynaud Thanks. While reviewing the release notes and each commit that mentioned verification keys, I overlooked the higlights **requires Mithril client (a.k.a mithril-client-cli) 0.5.7+ for restoration** which clearly mentioned the requirement.

I'll update the workflow in the guild-operators tools to check for new GA (non pre) release versions and kick off the builds so the scripts/tools stay in sync with the version being run on the network.

Thanks

**Comment by TrevorBenson** [comment]
> > Hi @TrevorBenson, you just need to use the newest `mithril-client` released in [`2347.0`](https://github.com/input-output-hk/mithril/releases/tag/2347.0) in order to restore the snapshot (the minimum version is now `0.5.5`) 👍
> 
> @jpraynaud Thanks. While reviewing the release notes and each commit that mentioned verification keys, I overlooked the higlights **requires Mithril client (a.k.a mithril-client-cli) 0.5.7+ for restoration** which clearly mentioned the requirement.
> 
> I'll update the workflow in the guild-operators tools to check for new GA (non pre) release versions and kick off the builds so the scripts/tools stay in sync with the version being run on the network.
> 
> Thanks

@jpraynaud I think the release notes may need adjustment. You mentioned 0.5.5, and downloading versoin 2347.0 confirms the client is **0.5.5+0780dfa**. However, the release notes mention a mithril-client 0.5.7+ requirement.

**Comment by jpraynaud** [comment]
> .7

Thanks @TrevorBenson you are completely right, we have fixed the release notes with minimum version `0.5.5` :+1:

**Comment by WesleyBatista** [comment]
Following the guides from https://hydra.family/head-protocol/docs/tutorial/?system=macos#step-1-connect-to-cardano currently leads to:
```
$ mithril-client cardano-db download latest
1/5 - Checking local disk info…
2/5 - Fetching the certificate and verifying the certificate chain…                                                                                                                                                                                 Jan 04 20:34:24.235 CRIT Could not create certificate from API message: unknown variant `next_protocol_parameters`, expected one of `snapshot_digest`, `cardano_transactions_merkle_root`, `next_aggregate_verification_key`, `latest_block_number` at line 1 column 2852.
Error: Can not verify the certificate chain from certificate_hash: '5dd6407b0cb135141bded99695fa99b838d111164a250c201ce16242938b4b80'

Caused by:
    unknown variant `next_protocol_parameters`, expected one of `snapshot_digest`, `cardano_transactions_merkle_root`, `next_aggregate_verification_key`, `latest_block_number` at line 1 column 2852
```
and [this comment](https://github.com/input-output-hk/mithril/issues/1375#issuecomment-1830343991) really helped!

I updated `mithril_version` from `2430.0` to the latest released version `2450.0`, and I could get passed the first step 🎉 ✅ 

Ideally, the version bump process should update that documentation section to ensure smooth onboarding and adoption. Nonetheless, thank you so much for the logging and troubleshooting of this issue, it was very helpful!

**Comment by jpraynaud** [comment]
Hi @WesleyBatista, thank you for pointing that out. 

I have created a PR on the Hydra repository https://github.com/cardano-scaling/hydra/pull/1775 that leverages the new one-line installer we have implemented for Mithril (which makes it easy to install the latest released version) :+1:
